src/FirstSteps/RPG/Game.cs

```diff
@@ -32,51 +25,63 @@ public static void CreateHero()
 
         }
-        public static void HandleCommand(string command)
+        private static void HandleCommand(string command)
         {
-            switch (command)
+            var adventure = _adventuresRegistry.GetAdventure(command);
+            adventure?.Enter(_hero);
+
+            if (command == "Stats")
             {
-                case "stats":
-                    DisplayHeroStats();
-                    break;
-                case "treasure adventure":
-                    _treasureChest.Enter(_hero);
-                    break;
-                case "inventory":
-                    _inventory.DisplayAllItems(_hero);
-                    break;
-                case "forest":
-                    _forest.Enter(_hero);
-                    break;
-                case "dungeons":
-                    _dungeons.Enter(_hero);
-                    break;
-                case "mine":
-                    _mine.Enter(_hero);
-                    break;
-                case "village":
-                    _village.Enter(_hero);
-                    break;
-                case "hell":
-                    _hell.Enter(_hero);
-                    break;
-                default:
-                    Display.WarningText($"Command {command} not recognized");
-                    break;
+                DisplayHeroStats();
             }
         }
         public static void DisplayHeroStats()
         {
             _hero.DisplayStats();
         }
 
-        public static void DisplayAllRaces()
+        private static void DisplayAllRaces()
         {
             var races = Enum.GetNames(typeof(Races));
             for (int i = 0; i < races.Length; i++)
             {
                 Console.WriteLine($"{i + 1}. {races[i]}");
             }
         }
+        private static string MenuSelect()
+        {
+            var menu = new SelectionPrompt<string>();
+
+            menu.AddChoice("Hero")
+                .AddChild("Stats");
+
+            menu.AddChoiceGroup("Adventures", _adventuresRegistry.GetCommands());
+            menu.AddChoice("Settings")
+                .AddChild("end");
+            return AnsiConsole.Prompt(menu);
+
+
+        }
+        public static void GameMenu()
+        {
+            AnsiConsole.MarkupLine($"{Emoji.Known.BowAndArrow} [darkgreen]Welcome to the RPG game [/] {Emoji.Known.CrossedSwords}");
+
+            Game.CreateHero();
+
+            AnsiConsole.MarkupLine($"{Emoji.Known.Dagger} [red]Let's play the game[/] {Emoji.Known.Dagger}");
+
+            Console.WriteLine("\nType 'end' if you want to quit.");
+            Console.WriteLine("\nIf You want go back to the previous menu press button\n");
+            Console.WriteLine("________________________________");
+            string userInput;
+            do
+            {
+                userInput = MenuSelect();
+                HandleCommand(userInput);
+                _adventuresRegistry.GetCommands();
```

Comment:
As I wrote in the comment above: `_adventuresRegistry.GetCommands(); - it is completely useless here, note that it returns the commands list but you didn't add the result to any variable`